libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -1231,6 +1204,41 @@ internal object MapboxRouteLineUtils {
         }
     }
 
+    /**
+     * Looks for a property in the [FeatureCollection] matching an identifier in the
+     * collection of [RouteStyleDescriptor] items and if a match is found the color from
+     * the [RouteStyleDescriptor] is returned. If no match is found the fallback color is returned.
+     */
+    internal fun getMatchingColors(
+        featureCollection: FeatureCollection?,
+        styleDescriptors: List<RouteStyleDescriptor>,
+        fallbackLineColor: Int,
+        fallbackLineCasingColor: Int
+    ): Pair<Int, Int> {
+        return styleDescriptors.firstOrNull {
+            featureCollectionHasProperty(featureCollection, 0, it.routeIdentifier)
+        }?.run {
+            Pair(this.lineColor, this.lineCasingColor)
+        } ?: Pair(fallbackLineColor, fallbackLineCasingColor)
```

Comment:
```suggestion
        } ?: LineColors(fallbackLineColor, fallbackLineCasingColor)
```
nit: introduce an internal structure to improve the readability

Reply:
Let's merge as is to avoid a CI cycle.